.github/workflows/multi-approvers.yml

```diff
@@ -40,8 +54,21 @@ jobs:
       contains(fromJSON('["pull_request", "pull_request_review"]'), github.event_name)
     runs-on: 'ubuntu-latest'
     steps:
-      - name: 'Download members.json'
-        id: 'download-members-json'
+      - name: 'Check inputs'
+        shell: 'bash'
+        run: |-
+          if [[ -z "${{ inputs.org-members-path }}" &&
+                -z "${{ inputs.members-url }}" &&
+                -z "${{ inputs.local-members-path }}" &&
+                -z "${{ inputs.team }}" ]]; then
+            echo "Invalid inputs. One of org-members-path, members-url, local-members-path, or team must be set." 1>&2
```

Comment:
Don't access inputs directly in a script (it's a shell injection vuln). Set inputs as envvars and access the envvars.

Reply:
Please fix before merging, regardless of the future. We cannot have known security exploits sitting in code at HEAD.